requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 pytest
 flake8
 autopep8
+multiversx_sdk_core>=0.4.1
```

Comment:
The tests depend on newer classes, that did not exist in `v0.3.*` (the production code does not depend on them, though).